src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { ALERT_SIZE, ALERT_TYPE, ALERT_ICON } from "./enums";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
```

Comment:
Waarom hebben de stories eigenlijk styling nodig? In de documentatie staat nu dat die dat niet nodig hebben. Ik vermoed dat dit gewoon weg mag.

Reply:
Deze is nodig voor de button styling binnen de alert. Anders krijgt die geen correcte styling als ik het zonder implementeer.

Reply:
Omdat er geen buttons in de alert zitten vraag ik mij eigenlijk af waarom de button styling in in de scss van de alert zit. Volgens mij moet die daar uit, als je de button wil gebruiken in een slot (net zoals elke andere component) moet je de button styling importeren, omdat die component zelf dat nodig heeft, maar de alert heeft dit niet nodig.